packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.test.tsx

```diff
@@ -23,18 +25,14 @@ jest.mock('./utils', () => {
         detectStorefrontPreview: jest.fn()
     }
 })
+jest.mock('../../auth/index.ts')
 
 describe('Storefront Preview Component', function () {
-    const oldWindow = window
-
     beforeEach(() => {
-        // eslint-disable-next-line
-        window = {...oldWindow}
+        delete window.STOREFRONT_PREVIEW
```

Comment:
Making sure that `window.STOREFRONT_PREVIEW` does not leak to other tests in this file.